src/WebJobs.Script.WebHost/Management/InstanceManager.cs

```diff
@@ -122,6 +140,38 @@ public bool StartAssignment(HostAssignmentContext context)
             }
         }
 
+        private ScmSpecialization.Type ScmSpecializationType(HostAssignmentContext context)
```

Comment:
Looks too complicated for what we are adding. :) 
We still only have 1 specialization type. Only change is error handling when blob doesn't exist.